internal/grafanadi/service/debugpodlist.go

```diff
@@ -0,0 +1,75 @@
+package service
+
+import (
+	"sort"
+	"time"
+
+	"github.com/alipay/container-observability-service/internal/grafanadi/model"
+	storagemodel "github.com/alipay/container-observability-service/pkg/dal/storage-client/model"
+)
+
+func ConvertPodyamls2Table(podyamls []*storagemodel.PodYaml) []*model.DebugPodListTable {
+	podyamlsSet := DeduplicateAndTopK(podyamls, 5)
+
+	tables := make([]*model.DebugPodListTable, 0)
+	for _, v := range podyamlsSet {
+		var state string = "在线"
```

Comment:
can you please use keyword in english for the state and please keep the naming consistent across functionalities, thanks :D

Reply:
fixed